compiler/importer.nim

```diff
@@ -13,12 +13,14 @@ import
   intsets, strutils, os, ast, astalgo, msgs, options, idents, lookups,
   semdata, passes, renderer, modulepaths, sigmatch, lineinfos
 
-proc readExceptSet*(c: PContext, n: PNode): IntSet =
+proc readExceptSet*(c: PContext, n: PNode, fromMod: PSym): IntSet =
   assert n.kind in {nkImportExceptStmt, nkExportExceptStmt}
   result = initIntSet()
   for i in 1 ..< n.len:
     let ident = lookups.considerQuotedIdent(c, n[i])
     result.incl(ident.id)
+    let s = strTableGet(fromMod.tab, ident)
```

Comment:
But this is wrong, ``import x except doesntExist`` is valid for forward and backward compatibility reasons. So the ``s`` can be nil here. And even then I don't really like this logic. You don't want to list "usages" for the symbol that is actually explicitly not used here.

Reply:
Okay I reverted the changes.